odoo/tools/translate.py

```diff
@@ -553,10 +553,20 @@ def __init__(self, source):
         self.source = pycompat.csv_reader(source, quotechar='"', delimiter=',')
         # read the first line of the file (it contains columns titles)
         self.fields = next(self.source)
+        self.fields += list(set(["imd_name", "imd_model"]) - set(self.fields))
 
     def __iter__(self):
         for entry in self.source:
-            yield zip(self.fields, entry)
+            res = dict(zip(self.fields, entry))
```

Comment:
Maybe convert `self.source` to a `csv.DictReader` and get rid of `self.fields` then?